src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -353,7 +353,7 @@ public IExternalControl Control
             DType ruleScope = null,
             bool forceUpdateDisplayNames = false,
             IExternalRule rule = null,
-            Features features = Features.None)
+            Features features = Features.DefaultFeatures)
```

Comment:
>Features.DefaultFeatures [](http://example.com/codeflow?start=32&length=24)

This is dangerous. I agree that we have to make this change, but it needs to be coordinated with PAClient.